### ISSUE TITLE
Low Wattage Reading

### ISSUE THREAD

**Issue by jimsl04** [Dfinitski/N7DDC-ATU-100-mini-and-extended-boards#32]

**Comment by jimsl04** [comment]
I completed the kit version.  The unit seems to function properly, but the Wattage readings don't correspond to the Wattage shown on the transmitter or when connected to another meter.  At 10+ Watts input the device shows ~5.6 Watts.  The unit will not function with an input wattage below 9+ Watts.

could I have wound the dual transformer incorrectly?

**Comment by Dfinitski** [comment]
Hi. 
Please count how many times a wire goes through core on the tandem-match.
Also, probably installed diodes are not recomended kind.
Check also resistive devider, 34k should be next to processor and 68k should be close to tandem-match.

**Comment by jimsl04** [comment]
The primary wires go through 1 time each.  the secondary wires go through ten times each.  Will check the divider asap.

**Comment by jimsl04** [comment]
I checked the Resistors.  All correct.  I couldn't tell on thee zeners.  However, While I was checking, it occurred to me that I have ten loops of wire on the outside of the cores, and thus eleven passes through the cores.  Is this enough to cause the problem?

**Comment by Dfinitski** [comment]
Yes. Should be 10 passes through the core, not 11.

сб, 2 янв. 2021 г. в 15:32, jimsl04 <notifications@github.com>:

**Comment by jimsl04** [comment]
Got it.  I'll correct and let you know the result.

**Comment by jimsl04** [comment]
Windings now correct.  Same problem exists.  Ideas?

**Comment by Dfinitski** [comment]
Please attach a foto of your tandem-match.

**Comment by jimsl04** [comment]
![match 1 of 2](https://user-images.githubusercontent.com/57305405/103471913-1d569980-4d4c-11eb-9363-cb963f8da6d3.jpg)
![match 2 of 2](https://user-images.githubusercontent.com/57305405/103471917-2f383c80-4d4c-11eb-9e6f-480a705621c8.jpg)

**Comment by Dfinitski** [comment]
Looks good.
Probably there are wrong diodes.
I don't have more ideas.

**Comment by jimsl04** [comment]
Right.  I'll just live with it (maybe swap out the diodes later).  I really like the device, and will tell others how great it is and how well you support it.  Thanks for your help.

Jim
